packages/@n8n/db/src/repositories/execution.repository.ts

```diff
@@ -1030,6 +1037,20 @@ export class ExecutionRepository extends Repository<ExecutionEntity> {
 				.andWhere('sw.projectId = :projectId', { projectId });
 		}
 
+		const isPostgres = this.globalConfig.database.type === 'postgresdb';
+
+		if (query.nodesExecuted && query.nodesExecuted.length > 0) {
+			const idConditions: string[] = [];
+			query.nodesExecuted.forEach((node, idx) => {
+				const fragment = isPostgres
```

Comment:
Did you get the changes to work in a postgresdb?
I had to change it as follows for the filtering to work
```
const fragment = isPostgres
					? `jsonb_exists(execution_data.data::jsonb->4, :nodeName${idx})`
					: `json_type(json_extract(execution_data.data, '$[4].' || :nodeName${idx})) IS NOT NULL`;
```
Is this related to different postgres versions?